### ISSUE TITLE
The parsec website is down, do you have the datasets for PARSEC3 so we can upload them somewhere?

### ISSUE THREAD

**Issue by LepingLi0810** [cirosantilli/parsec-benchmark#18]
Sorry for the bother. Princton closed the PARSEC documentation site, so there is no way to get the dataset to run the benckmark, especially for splash2x. Is it possible that you can upload a copy of the dataset? Thanks so much for porting PARSEC to latest Ubuntu. You are a true hero.

**Comment by cirosantilli** [comment]
OMG, loooool. I wonder if it is permanent though or just a bug. archive.org seems to have all the zips:

- http://web.archive.org/web/20230902151148/http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-core.tar.gz
- http://web.archive.org/web/20160809002339/http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-input-sim.tar.gz
- http://web.archive.org/web/20160809000537/http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-input-native.tar.gz

I'll also try to upload them to GitHub once download finishes.

**Comment by LepingLi0810** [comment]
Thanks so much!!! You make this world better.

**Comment by cirosantilli** [comment]
The two smaller downloads from web archive work but the big one fails half way. I also tried `wget -c` as per https://unix.stackexchange.com/questions/165875/resume-failed-download-using-linux-command-line-tool but that didn't help either, it then immediately fails to progress with:

```
$ wget -c http://web.archive.org/web/20160809000537/http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-input-native.tar.gz
--2023-11-21 19:46:55--  http://web.archive.org/web/20160809000537/http://parsec.cs.princeton.edu/download/3.0/parsec-3.0-input-native.tar.gz
Resolving web.archive.org (web.archive.org)... 207.241.237.3                                                                                                                                  
Connecting to web.archive.org (web.archive.org)|207.241.237.3|:80... connected.                                                                                                               
HTTP request sent, awaiting response... 206 Partial Content                                    
Length: 2429601226 (2.3G), 1415303570 (1.3G) remaining [application/x-gzip]                    
Saving to: ‘parsec-3.0-input-native.tar.gz’                                                    
                                                                                                                                                                                              
parsec-3.0-input-native  41%[++++++++++++                   ] 967.31M  --.-KB/s    in 3.8s    
                                                                                               
2023-11-21 19:47:06 (0.00 B/s) - Connection closed at byte 1014297656. Retrying.
```

so I've asked a collaborator of this repo to share a copy he had on his computer and he handed it to me and I'm uploading them all to: https://github.com/cirosantilli/parsec-benchmark/releases/tag/3.0
